samples/matter/common/src/bridge/Kconfig

```
@@ -48,4 +48,32 @@ config BRIDGE_BT_MINIMUM_SECURITY_LEVEL
 	   allowed to connect to the bridge. The level cannot be set to a value smaller than 2, as it would mean that
 	   the communication with the connected devices will not be encrypted, what is not considered to be secure.
 
+config BRIDGE_BT_SCAN_WINDOW
+	int "Time duration of a central actively scanning for devices withing scan interval, it is equal BRIDGE_BT_SCAN_WINDOW*0.625 ms"
+	default 40  if NET_L2_OPENTHREAD
+	default 48
+
+config BRIDGE_BT_SCAN_INTERVAL
+	int "Time between consecutive Bluetooth scan windows, it is equal BRIDGE_BT_SCAN_INTERVAL*0.625 ms"
+	default 160 if NET_L2_OPENTHREAD
```

Probably you could add similar comment in case of settings different defaults for Thread.
```suggestion
	default 160 if NET_L2_OPENTHREAD # It is necessary to decrease Bluetooth LE activity to leave some radio time to Thread
```